plugins/dynatrace/src/api/DynatraceClient.ts

```diff
@@ -52,11 +57,31 @@ export class DynatraceClient implements DynatraceApi {
     );
   }
 
+  async getDynatraceSyntheticFailures(
+    syntheticsId: string,
+  ): Promise<DynatraceSyntheticResults | undefined> {
+    if (!syntheticsId) {
+      throw new Error('Dynatrace syntheticId is required');
+    }
+
+    return this.callApi(`synthetic/execution/${syntheticsId}/FAILED`, {});
+  }
+
+  async getDynatraceSyntheticLocationInfo(
+    syntheticLocationId: string,
+  ): Promise<DynatraceSyntheticLocationInfo | undefined> {
+    if (!syntheticLocationId) {
+      throw new Error('Dynatrace syntheticLocationId is required');
+    }
+
+    return this.callApi(`synthetic/locations/${syntheticLocationId}`, {});
+  }
+
   async getDynatraceProblems(
     dynatraceEntityId: string,
   ): Promise<DynatraceProblems | undefined> {
     if (!dynatraceEntityId) {
-      throw new Error('Dynatrace entity ID is required');
+      throw new Error('Dynatrace entity Id is required');
```

Comment:
Really? :) Feels like `ID` or `id` would be the good ones, while `Id` is the least good option